helpers/utils.js

```diff
@@ -177,12 +189,7 @@ exports.performAction = async function performAction(request, action) {
         const pageId = page.target()._targetId;
 
         try {
-            const response = await action(page, request);
-            return {
-                ...response,
-                contextId,
-                pageId,
-            };
+            return await action(page, request);
```

Comment:
Maybe we can perform ids assignment and closePage handling for all actions here?
`formResponse` can just collect html and cookies after wait (it should be renamed then)

Reply:
Yeah, that's what I'm thinikng about. `formResponse` does too many actions at once. Probably, we should divide it into several parts.
After this, `performAction` would close the page and get Ids.
I will deal with that.